Move OpInfos for torch.fft into `opinfo.definitions`

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #83554
* #83547
* #83563
* #83560
* #83559
* #83277
* __->__ #83276

Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

peterbell10 committed:
Move OpInfos for torch.fft into `opinfo.definitions`

Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

[ghstack-poisoned]

albanD reviewed (approved):
Sorry for the delay on this.
Can you do a rebase before merging to reduce chances of merge conflicts please?

peterbell10 committed:
Rebase on "Move OpInfos for torch.fft into `opinfo.definitions`"


Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

[ghstack-poisoned]

peterbell10 committed:
Update on "Move OpInfos for torch.fft into `opinfo.definitions`"


Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

[ghstack-poisoned]

peterbell10 committed:
Update on "Move OpInfos for torch.fft into `opinfo.definitions`"


Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

[ghstack-poisoned]

peterbell10 committed:
Update on "Move OpInfos for torch.fft into `opinfo.definitions`"


Ref #82518

This moves the `op_db` entries into `opinfo/definitions/fft.py` and
also appends them to `common_methods_invocations.op_db` so existing
users are unaffected by this change.

[ghstack-poisoned]
